consensus/types/src/beacon_block.rs

```diff
@@ -627,7 +632,8 @@ impl<E: EthSpec, Payload: AbstractExecPayload<E>> BeaconBlockElectra<E, Payload>
             body: BeaconBlockBodyElectra {
                 proposer_slashings: base_block.body.proposer_slashings,
                 attester_slashings: base_block.body.attester_slashings,
-                attestations: base_block.body.attestations,
+                // TODO(eip7549) is setting to default ok here?
+                attestations: <_>::default(),
```

Comment:
No, that is not representative of the maximum size. You need to create a list with an electra attestation of maximum size